test/terra/backends/qasm_simulator/qasm_measure.py

```diff
@@ -195,3 +196,32 @@ def test_measure_nondeterministic_multi_qubit_without_sampling(self):
         self.assertSuccess(result)
         self.compare_counts(result, circuits, targets, delta=0.05 * shots)
         self.compare_result_metadata(result, circuits, "measure_sampling", False)
+
+class QasmMPSMeasureAlgorithms:
+    """QasmSimulator MPS algorithms for measure tests."""
+
+    SIMULATOR = QasmSimulator()
+    BACKEND_OPTS1 = {"mps_sample_measure_algorithm":"mps_apply_measure"}
+    BACKEND_OPTS2 = {"mps_sample_measure_algorithm":"mps_probabilities"}
+
+    # ---------------------------------------------------------------------
+    # Test MPS algorithms for measure
+    # ---------------------------------------------------------------------
+    def test_mps_measure_alg_qv(self):
```

Comment:
I am not sure where would be most suitable for this test. Perhaps under https://github.com/Qiskit/qiskit-aer/blob/main/test/terra/backends/aer_simulator/test_options.py? on the other hand, it is testing measurement, so that's why I put it in `test_measure`.
It's your call where to put it.

Reply:
Your last comment somehow disappeared. You mentioned putting the option `apply_measure` in `AerSimulator` and in `QasmSimulator`. This option is not new. It is already in both these files.